salesforce_api/kafka-salesforce-pricebook_update/config.toml

```diff
@@ -0,0 +1,7 @@
+salesforceBaseUrl = ""
+
+[choreo.kafka_salesforce_pricebook_update.salesforceOAuthConfig]
+clientId = ""
+clientSecret = ""
+refreshToken = ""
+refreshUrl = ""
```

Comment:
Missing new line

Reply:
Removed the file in [febd7ae](https://github.com/ballerina-guides/integration-samples/pull/31/commits/febd7ae79564c8b1f5f2e757f98d219c50d18857)